config/NuElectronPXSec.xml

```diff
@@ -14,8 +14,10 @@ WeinbergAngle  double   No                                 CommonParam[WeakInt]
 
   <param_set name="Default"> 
     <param type="string"  name="CommonParam"> WeakInt  </param>
-    
+    <param type="int"    name ="N-Integration-Samples" >              2000  </param>
```

Comment:
Ideas here. Would it be interesting to have the possibility to set the variance of the average in relative terms instead of the number of iterations? It probably makes more sense, would you be able to implement this? we can have a chat if you prefer.